### ISSUE TITLE
Synonims handling error

### ISSUE THREAD

**Issue by Mangirdas** [rodrigopivi/Chatito#58]
Thanks for creating chatito.

There is entities synonim handling bug in rasa output, which happened going from chatito v1 to v2.
Different synonim values are not mapped to same value for entities.
Chatito 1.2.2 was correct behaviour.

Chatito file:

```
%[test-intent]('training': '5', 'testing': '1')
    ~[going?] @[city]

@[city]
    ~[nyc]

~[nyc]
    new york
    new york city
    nyc

~[going]
    go to
    go
```

Chatito 2.1.5:
```
	{"rasa_nlu_data":{"regex_features":[],

	"entity_synonyms":[{"synonyms":["new york city","new york"],"value":"nyc"}],

	"common_examples":
		[
		{"text":"asking new york city","intent":"test-intent","entities":[{"end":20,"entity":"city","start":7,"value":"new york city"}]},
		{"text":"new york","intent":"test-intent","entities":[{"end":8,"entity":"city","start":0,"value":"new york"}]},
		{"text":"new york city","intent":"test-intent","entities":[{"end":13,"entity":"city","start":0,"value":"new york city"}]},
		{"text":"asking new york","intent":"test-intent","entities":[{"end":15,"entity":"city","start":7,"value":"new york"}]},
		{"text":"asking nyc","intent":"test-intent","entities":[{"end":10,"entity":"city","start":7,"value":"nyc"}]}
		]}}
```

Chatito 1.2.2:
```
	{
    "rasa_nlu_data": {
        "regex_features": [],
        "entity_synonyms": [
            {
                "value": "nyc",
                "synonyms": [
                    "new york",
                    "new york city"
                ]
            }
        ],
        "common_examples": [
            {
                "text": "go new york city",
                "intent": "testintent",
                "entities": [
                    {
                        "start": 3,
                        "end": 16,
                        "value": "nyc",
                        "entity": "city"
                    }
                ]
            },
            {
                "text": "go to new york",
                "intent": "testintent",
                "entities": [
                    {
                        "start": 6,
                        "end": 14,
                        "value": "nyc",
                        "entity": "city"
                    }
                ]
            },
```

Chatito 1.2.2 was correct behaviour. As in here - https://rasa.com/docs/nlu/0.12.1/dataformat/#entity-synonyms

**Comment by Mangirdas** [comment]
@rodrigopivi any plans on this?

**Comment by rodrigopivi** [comment]
Hi @Mangirdas, i'm not very updated with all Rasa's new features but i think Rasa should be able to handle this use case correctly, can you confirm if this affect training the model or its output? Thanks

**Comment by Mangirdas** [comment]
@rodrigopivi model trains, but model trains incorrectly.

With Chatito >=2.0 RASA model does not learn that nyc/new york is same thing. It thinks that nyc/new york is different entity values without any relation to each other. Model does not generalise.

I can confirm 1.2.2 was correct behaviour. Is this difficult fix?

**Comment by rodrigopivi** [comment]
Fixed in v2.2.0, thanks!
